src/components/Filter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { STATUSES, AllProductRequestProps, ProductRequest } from 'types';
+import '../index.scss';
+
+interface IProps {
+  productRequests: ProductRequest[];
+  showData: ProductRequest[];
+  setShowData: any;
+}
+
+export const Filter = (props: IProps) => {
+  const { productRequests } = props;
+  const { showData } = props;
+  const { setShowData } = props;
```

Comment:
Keep in mind you can destructure more than one thing in a single line. This syntax is preferred:
```
const { productRequests, showData, setShowData} = props;
```

Reply:
https://dmitripavlutin.com/javascript-object-destructuring/

Reply:
Updated!